routes/notifications.js

```diff
@@ -22,7 +22,6 @@ router.get("/findById/:notificationId", async (req, res) => {
     });
     res.status(200).json(notifications);
   } catch (err) {
-    console.log(err);
```

Comment:
I discourage removing logging... This will make debugging very difficult regardless if its in development or production settings... If a request fails, we'd have no idea what went wrong or whether it should be something we need to look into.

Reply:
The code is supposed to be production ready so the TAs will be grading it. We cant leave logs anymore. We're not doing Dev anymore or any new features so nothing should really break. We can add logs while doing dev but must be removed for submission.

Reply:
If there is a network error and the code goes to the catch block. We'd have no idea if something went wrong in the db, our code, or network without logging. Even in production environment, things like this happen. Removing logging will make our code impossible to maintain.